### PR TITLE
Task.bar 47

### PR DESCRIPTION
**Pull request by stefanomusicman**
# Description of the problem
created UI for adding ings in panel on search by ingredients screen

## How does it solve the problem

# References (also the Jira ticket)
BAR-47
Untitled

# Checklist:

- [x] I re-reviewed the Jira ticket now to make sure 
- [x] My code follows the style guidelines of this project & of Dart
- [x] I have performed a self-review of my own code
- [x] I have commented my code in hard-to-understand areas
- [x] I have wrote some documentation if I changed a big behavior of the code
- [x] My changes generate no new warnings/and if they do, I described them to the team
- [ ] I have added tests that prove my fix is effective or that my feature works if I worked on a logic class/function
- [ ] New and existing unit and integration tests pass locally with my changes

### CODE REVIEW

**Comment by danielemonaca** [comment]
Nice!
The image will be changed because it is not supposed to be a CIG but an ingredient image, but that will come later

File: lib/ui/screens/searchByIngs/custom_card_cig.dart [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+import '../../../cig/CIG.dart';
+
+class CustomCardCIG extends StatelessWidget {
```
Comment (danielemonaca) [inline lib/ui/screens/searchByIngs/custom_card_cig.dart:4]: This probably shouldn't exist, as there should be no CIG in search by Ings

**Comment by stefanomusicman** [comment]
Untitled

**Comment by danielemonaca** [comment]
Could you increase the size of the image? Towards the upper side

**Review by danielemonaca — APPROVED** [review]

**Pull request merged**